epochStart/metachain/trigger.go

```diff
@@ -238,11 +238,21 @@ func (t *trigger) SetFinalityAttestingRound(round uint64) {
 }
 
 // Revert sets the start of epoch back to true
-func (t *trigger) Revert(round uint64) {
+func (t *trigger) Revert(header data.HeaderHandler) {
+	if check.IfNil(header) || !header.IsStartOfEpochBlock() || header.GetEpoch() == 0 {
```

Comment:
Shouldn't it be possible to revert a StartOfEpochBlock?

Reply:
reverting a trigger should happen only if header is of type start of epoch block.

Reply:
ok